servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2KeepAlivePolicies.java

```diff
@@ -30,7 +34,7 @@ public final class H2KeepAlivePolicies {
     static final KeepAlivePolicy DISABLE_KEEP_ALIVE =
             new DefaultKeepAlivePolicy(ofDays(365), ofDays(365), false);
     static final Duration DEFAULT_IDLE_DURATION = ofSeconds(30);
-    static final Duration DEFAULT_ACK_TIMEOUT = ofSeconds(30);
+    static final Duration DEFAULT_ACK_TIMEOUT = ofSeconds(5);
```

Comment:
Consider adding the default value as information to a javadoc block (config builder and/or config)

Reply:
👍